<launch>
    <!-- 实验节点 -->
    <node pkg="nav_goal" type="my_voice_nav" name="my_voice_nav" output="screen"/>
    <!-- 打开语音采集节点 -->
    <node name="voice_collect" pkg="robot_audio" type="voice_collect_node" output="screen">
        <!-- 音频文件目录 -->
        <param name="audio_file" type="string" value="./AIUI/audio/audio.wav "/>
    </node>
    <!-- 打开语音服务 -->
    <node pkg="robot_audio" type="voice_aiui_node" name="voice_aiui_node"/>
</launch>